tests/test_evented_model.py

```diff
@@ -845,3 +848,52 @@ class Model(EventedModel):
         m.a = 3
     check_mock.assert_has_calls([call(Model, "a", 3, 1)])
     mock1.assert_called_once()
+
+
+def test_if_event_is_emited_only_once():
```

Comment:
Sorry, one last spelling nit.

```suggestion
def test_if_event_is_emitted_only_once():
```